support-frontend/webpack.dev.js

```diff
@@ -4,7 +4,7 @@ const common = require('./webpack.common.js');
 
 module.exports = merge(common('[name].css', '[name].js', false), {
   mode: 'development',
-  devtool: 'inline-source-map',
+  devtool: 'cheap-module-eval-source-map',
```

Comment:
For future people who might see this PR I'm going to link to the documentation on this, because I had to look it up.